Makefile

```diff
@@ -5,7 +5,10 @@ all: build
 # Install some Quicklisp dependencies.
 ql-deps:
 	# 2023-03: we want str:ensure-suffix, not yet in Quicklisp.
-	git clone https://github.com/vindarel/cl-str/ ~/quicklisp/local-projectsi
+	git clone https://github.com/vindarel/cl-str/ ~/quicklisp/local-projects/cl-str
+	# 2023-05: log4cl doesn't compile on newer versions of SBCL
```

Comment:
Add an example SBCL version with this issue (yours: 2.3.4) and/or a link to your Github issue and it's perfect.

Reply:
Done, i've added the version and link to issue in the the comments.